### DISCUSSION TITLE
yaml的随机字符串(rd)？不随机的问题？

### DISCUSSION THREAD

**Discussion by vr-yan** [IceTiki/ruoli-sign-optimization#125] [Q&A]
yml配置
  forms:
  - form:
      title: 晨检体温
      value: {"str+":"<rd>36.1\a36.2\a36.3</rd>°", "flag":"tf"}
      number: 1
  - form:
      title: 午检体温
      value: {"str+":"<rd>36.3\a36.4\a36.5</rd>°", "flag":"tf"}
      number: 2
  - form:
      title: 晚检体温
      value: {"str+":"<rd>36.2\a36.3\a36.4</rd>°", "flag":"tf"}
      number: 3
输出日志
>>『每日晨午晚检健康签到』第1个配置出错了
表单未匹配到你设置的值：『<rd>36.136.236.3</rd>°』
，你上次/系统选的值为：『』
--------------------------------------
是只能提交填空信息的时候用吗？
签到图片↓
https://img1.imgtp.com/2022/10/15/zBkbHaTx.jpg

**Comment by IceTiki** [comment]
格式错了。
`value: {"str+":"<rd>36.2\a36.3\a36.4</rd>°", "flag":"tf"}`
